frontend/src/components/StatisticsModal.jsx

```diff
@@ -11,70 +11,140 @@ import DialogTitle from "@mui/material/DialogTitle";
 import Box from "@mui/material/Box";
 import Grid from "@mui/material/Grid";
 import Button from "@mui/material/Button";
+import { Stack } from "@mui/system";
 
 //eslint-disable-next-line react/prop-types
 function StatisticsModal({ modalOpen, closeModal }) {
-	const [stats, setStats] = useState({});
+	// const [stats, setStats] = useState({});
+	// const [siteStatistics, setSiteStatistics] = useState({});
+	const [sites, setSites] = useState({});
+	const [sitePointCount, setSitePointCount] = useState({});
+	const [catalogueMaterialPercentages, setCatalogueMaterialPercentages] =
+		useState({});
 
 	useEffect(() => {
 		/**
 		 * Gets statistics from aggregateStatisticsController.js and places in the state for the component
 		 */
-		async function statsGetter() {
-			try {
-				const response = await http.get(
-					"/aggregateStatisticsGenerators/catalogue/1",
-				);
-				log.debug("Response data before setStats(): " + response.data);
-				setStats(response.data);
-				log.debug(
-					"stats variable after setState(response.data) is called: " + stats,
-				);
-			} catch (error) {
-				log.error("Error fetching statistics: ", error);
-			}
-		}
-		statsGetter();
-	}, [stats]);
+
+		http
+			.get("/aggregateStatisticsGenerators/catalogue/1")
+			.then((response) => {
+				const catalogueStatistics = response.data;
+				let catalogueMaterialPercentagesStorage = new Array();
+				let count = 0;
+				for (const [key, value] of Object.entries(
+					catalogueStatistics["Material Data"]["Material Percentages"],
+				)) {
+					log.info("SM 40 Percentage: " + value);
+					log.info("SM 41 Percentage: " + key);
+					const newDataPoint = {
+						id: count,
+						value: value,
+						label: key,
+					};
+					count++;
+					catalogueMaterialPercentagesStorage.push(newDataPoint);
+				}
+				setCatalogueMaterialPercentages(catalogueMaterialPercentagesStorage);
+			})
+			.catch((error) => {
+				console.error("Error fetching site statistics:", error);
+			});
+
+		http
+			.get("/sites")
+			.then((response) => {
+				const sitesResponse = response.data;
+				setSites(sitesResponse);
+				let sitePointCounts = new Array();
+				sites.forEach((currentSite) => {
+					const newDataPoint = {
+						id: currentSite.id,
+						value: currentSite.artifacts.length,
+						label: currentSite.name,
+					};
+					sitePointCounts.push(newDataPoint);
+				});
+				setSitePointCount(sitePointCounts);
+			})
+			.catch((error) => {
+				console.error("Error fetching all sites.", error);
```

Comment:
Use multi-level logger `log` instead of `console`

Reply:
Fixed